docs/pages/bssw/bssw_styling_common.md

```diff
@@ -62,15 +63,34 @@ The main body is the portion of the article below the deck. BSSw.io uses [Github
 Images for content are stored in the main `bssw.io`repository in the `images/` subdirectory.
 To reference the images in the `*.md` article file, copy the image file is to the `images/` directory and then:
 
-- Add the relative path to the image file from the `*.md` file. Using the "logo" class helps constrain the size of the image.
+- Add the relative path to the image file from the `*.md` file. 
 ````
-<img src='../../images/YOUR-IMAGE-NAME.png' class='logo'/>
+<img src='../../images/YOUR-IMAGE-NAME.png' class='logo'/>[optional caption]
 ````
-- Please ensure we have permission to use the logo/image 
+- Please ensure we have permission to use the logo/image
 - Please ensure logo is clear and high resolution
 - Please remember to commit the image file with the `*.md` file on your git branch
+- Use the `logo` class to constrain the rendered size of the image.
+- Use the `page` class to prevent enlargement (recommended for horizontal images under 1000 px wide).
+- Use the `page lightbox` class for vertical images and images that have details that could be better viewed if the image were expanded to fill the screen.
+- Add an image credit as appropriate.
+- Captions are optional (and in general, mildly discouraged) and must immediately follow the img tag if used.
 
-See  [images/README.md](https://github.com/betterscientificsoftware/bssw.io/blob/master/images/README.md) for more details.
+Additional points about images and captions:
+
+- Captions should not include hyperlinks.  They are undesirable from the design standpoint: it disrupts reading the text (your eye jumps straight to the link) and any critical links would be best placed in the story itself.  Further, hyperlinks prevent the front-end from actually interpreting the caption as a caption.
```

Comment:
Just an observation that this remark I think applies potentially to *all text* in an article...not just a caption though its probably more of an issue for captions. My reason for pointing this out is that it is one of the reasons for introducing and using wikipedia style footnoting that `wikize_refs.py` is designed to support.

Reply:
True, but something different and should be documented elsewhere.  The requested format for many image credits will include a link -- natural these days.  They *break* the processing of captions somehow, so we *can't* use them in captions.  Not just discouraged.